src/PaperG/FirehoundBlob/Facebook/Validators/UnmanagedFacebookBlobValidator.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace PaperG\FirehoundBlob\Facebook\Validators;
+
+use PaperG\FirehoundBlob\Facebook\UnmanagedFacebookBlob;
+use PaperG\FirehoundBlob\JsonValidator;
+
+class UnmanagedFacebookBlobValidator extends JsonValidator
+{
+    const RELATIVE_PATH = '/../../Schema/Facebook/unmanagedFacebookBlobCreate.json';
```

Comment:
Hmm, good point, I should probably point at the generic, though it's kind of meaningless...